packages/components/src/styles/components/tag.scss

```diff
@@ -15,6 +15,7 @@ $hds-tag-border-radius: 50px;
 .hds-tag {
   display: inline-flex;
   align-items: stretch;
+  width: fit-content;
```

Comment:
You should test if it could possibly exceed the width of its container with this style set. May want to add `max-width: 100%`

Reply:
I did a bit of quick testing and it doesn't seem very likely that the tag would overflow it's container but it _is_ possible with containers that have very narrow widths. So you _could_ add `.hds-tag, .hds-tag__link {max-width: 100%;}` to be extra cautious although it could be overkill so up to you...

Reply:
Thanks, just be sure to test yourself to confirm the effect as I just did quick in browser tests.